example/server.py

```diff
@@ -1,11 +1,25 @@
+import pickle
+
 from bottle import Bottle, redirect, run
+import datastore.core
 
 from switchboard import operator, configure
 from switchboard.middleware import SwitchboardMiddleware
 from switchboard.admin import app as switchboard
 
-configure()
 
+# Setup a file-based datastore with pickle serialization.
+import datastore.filesystem
+import os
+base_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
flake8 doesn't actually complain about this... I wonder if it's because I'm assigning variables below the imports and not defining a function or class. I know flake8 complains in those cases. Regardless, if possible, I'd like to keep the imports with the code for easy experimentation with different datastores.